.env.test

```diff
@@ -2,6 +2,7 @@ NODE_ENV=test
 PORT=4000
 APP_NAME=kaws-test
 DD_APM_ENABLED=false
+APP_URL=https://foobar.com
```

Comment:
Might be clearer to use the real url here, I'm assuming we make no actual requests in tests.